frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/Maskinporten.tsx

```diff
@@ -18,19 +19,23 @@ export const Maskinporten = (): React.ReactElement => {
     return <StudioSpinner spinnerTitle={t('general.loading')} />;
   }
 
-  if (isLoggedInWithAnsattporten) {
-    return <div>View when logged in comes here</div>;
-  }
-
   return (
     <TabContent>
       <StudioHeading level={2} size='sm' spacing>
         {t('settings_modal.maskinporten_tab_title')}
       </StudioHeading>
-      <StudioParagraph spacing>{t('settings_modal.maskinporten_tab_description')}</StudioParagraph>
-      <StudioButton onClick={handleLoginWithAnsattporten}>
-        {t('settings_modal.maskinporten_tab_login_with_ansattporten')}
-      </StudioButton>
+      {!isLoggedInWithAnsattporten && (
```

Comment:
What do you think about creating a MaskinportenPageTemplate? This allows us to maintain the heading and avoid "negative (not) isLoggedIn" logic.

MaskinportenPageTemplate:
```tsx
const MaskinportenPageTemplate = ({ children }) => {
    return (
        <TabContent>
            <StudioHeading level={2} size='sm' spacing>
                {t('settings_modal.maskinporten_tab_title')}
            </StudioHeading>
            {children}
        </TabContent>
    )
}
```


This way, it can be used like this:

```tsx
import React, { type ReactElement } from 'react';
import { useTranslation } from 'react-i18next';
import { TabContent } from '../../TabContent';
import { StudioButton, StudioHeading, StudioParagraph, StudioSpinner } from '@studio/components';
import { useIsLoggedInWithAnsattportenQuery } from 'app-development/hooks/queries/useIsLoggedInWithAnsattportenQuery';
import { ScopeList } from './ScopeList';

export const Maskinporten = (): ReactElement => {
    const { data: isLoggedInWithAnsattporten, isPending: isPendingAuthStatus } =
        useIsLoggedInWithAnsattportenQuery();

    const { t } = useTranslation();

    const handleLoginWithAnsattporten = (): void => {
        console.log('Will be implemented in next iteration when backend is ready');
    };

    if (isPendingAuthStatus) {
        return <StudioSpinner spinnerTitle={t('general.loading')} />;
    }

    if (isLoggedInWithAnsattporten) {
        return (
            <MaskinportenPageTemplate>
                <ScopeList />
            </MaskinportenPageTemplate>
        )
    }

    return (
        <MaskinportenPageTemplate>
            <StudioParagraph spacing>
                {t('settings_modal.maskinporten_tab_description')}
            </StudioParagraph>
            <StudioButton onClick={handleLoginWithAnsattporten}>
                {t('settings_modal.maskinporten_tab_login_with_ansattporten')}
            </StudioButton>
        </MaskinportenPageTemplate>
    );
};
```